src/app/workflow/dag/dag.component.html

```diff
@@ -49,7 +57,7 @@
       </div>
 
       <div *ngIf="dagType === 'classic' && !(isNFL$ | async)">
-        <mat-card class="alert alert-warning" role="alert" *ngIf="!(dagResult$ | async) && !selectedVersion?.valid">
+        <mat-card class="alert alert-warning" role="alert" *ngIf="!(dagResult$ | async) || !selectedVersion?.valid">
```

Comment:
Why did you change this? Not saying it's wrong, but please explain.

 It was a sort of confusing logic (I think I wrote it), but I think the idea was that it would only display the invalid descriptor message if the selected version was not valid. The way you've changed it, I believe it will now say a descriptor could not be found, even if there is a descriptor -- sometimes there are errors generating the DAG that are unrelated to the version not being valid.

Reply:
I misinterpreted this! My understanding was that an error message should display if at _least_ one check failed. I'll revert back